airbyte-webapp/src/pages/SourcesPage/pages/CreateSourcePage/components/SourceForm.tsx

```diff
@@ -54,10 +54,9 @@ const SourceForm: React.FC<IProps> = ({ onSubmit, sourceDefinitions, error, hasS
       afterSelectConnector();
     }
 
-    analyticsService.track("New Source - Action", {
-      action: "Select a connector",
-      connector_source_definition: connector?.name,
-      connector_source_definition_id: sourceDefinitionId,
+    trackNewSourceAction("Select a connector", {
+      connector_source: connector?.name,
```

Comment:
The PM's spec for these events calls this property `connector_source`. Not all source events are using this (yet) but at least it makes these actions consistent.